controller/src/beerocks/bml/bml_defs.h

```diff
@@ -610,6 +613,42 @@ struct BML_NEIGHBOR_AP {
     uint32_t ap_ChannelUtilization;
 };
 
+struct BML_CLIENT_CONFIG {
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_initial_radio;
+    // 1 for true, 0 for false, -1 for "not configured".
+    int stay_on_selected_device;
+    // comma-separated selected bands [2.4,5H,5L,6].
+    char selected_bands[BML_SELECTED_BANDS_LEN];
+    // Optional parameter
+    // The timelife delay of the client configuration (if configured), otherwise 0.
+    uint32_t time_life_delay_days;
```

Comment:
Remove this parameter, it should not be set per client, move it to conf file or UCI DB, this is not runtime configuration.

Reply:
if you want to give the client the ability to read it at least then it has to be here

Reply:
No need, he can read it from conf file

Reply:
I'll remove it from the BML_CLIENT_CONFIG, which is used in the "set" API
But keep it in the BML_CLIENT, which is used in the "get" API
Making it virtually as a "read only"

Reply:
@yaniv-yitzhak `time_life_delay_days`  parameter shouldn't  be part of conf file from these reasons:
1. we would need to handle it on boot.
2. no future ability to configure it per client.

either keep it in bml for read only or remove it completely and use define for aging.

Reply:
there are 2 layers here:
1. global params used by default (defined in prplmesh **config** file):
1.1. max lifetime delay - used for regular clients
1.2. unfriendly max lifetime delay - used for unfriendly clients
2. per-client (can be defined **per-client** in the **prplmesh-db**):
2.1 per-client lifetime delay - it is optional - if set it is used instead of the global parameters

- if we wish to support per-client configuration we must have both
- although we don't provide BML interface to configure the **global params** - user can edit them via UCI and restart prplmesh - so it is good to have them in **prplmesh config** file

Reply:
> 1. global params used by default (defined in prplmesh **config** file):
>    1.1. max lifetime delay - used for regular clients
>    1.2. unfriendly max lifetime delay - used for unfriendly clients

didn't see this mentioned in the HLD document

> * although we don't provide BML interface to configure the **global params** - user can edit them via UCI and restart prplmesh - so it is good to have them in **prplmesh config** file

it that case you need to add these parameters to aging handling mechanism on boot- not sure we have time for that.

I prefer to give access to less parameters as possible at least for now.

Reply:
I don't think there should be aging to these parameters - it is platform configuration just like any other prplmesh-config parameters

aging is only for clients' data/config - not for general platform configurations